rawfile/bd2fs.py

```diff
@@ -94,10 +109,11 @@ def NodeStageVolume(self, request, context):
 
         self.bds.NodePublishVolume(bd_publish_request, context)
 
-        mount_path = f"{request.staging_target_path}/mount"
-        Path(mount_path).mkdir(exist_ok=True)
-        be_formatted(dev=bd_publish_request.target_path, fs=get_fs(request))
-        be_mounted(dev=bd_publish_request.target_path, mountpoint=mount_path)
+        if get_access_type(request) == "mount":
```

Comment:
This one could be like the previous one :)

Reply:
How do you mean? Here it's just one if